CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.180.0, 17 March 2022
+
+- Bump to `go` `1.18` (@jeffwidman) [#4853](https://github.com/dependabot/dependabot-core/pull/4853)
+- Fix Bundler 1 spec failures (@deivid-rodriguez) [#4869](https://github.com/dependabot/dependabot-core/pull/4869)
+- Require Ruby 2.7 (@deivid-rodriguez) [#4864](https://github.com/dependabot/dependabot-core/pull/4864)
+- Bugfix: Include space in regex capturing group (@jeffwidman) [#4868](https://github.com/dependabot/dependabot-core/pull/4868)
+- Get bundler specs green when run on Ruby 3.1 (@deivid-rodriguez) [#4841](https://github.com/dependabot/dependabot-core/pull/4841)
+
```

Comment:
Missed one?

```suggestion
- NuGet version comparison fix [#4862](https://github.com/dependabot/dependabot-core/pull/4862)
```

Reply:
This changelog is autogenerated from the merge commits since the last release, it's weird that it missed this 😕

Reply:
It's probably because I squashed and it looks for merge commits. I'll stick to merge commits so the script keeps working!

Reply:
Yep, that'd explain it!